sites/playground/src/routes/tests/drag-to-reorder.svelte

```
@@ -4,25 +4,78 @@
 
     function handleReorder(newItems: string[]) {
         items = newItems;
     }
+
+    function handleObjectReorder(newItems: ReorderItem[]) {
+        newItems.forEach((item, order) => {
+            item.order = order;
+        });
+    }
+
+    const orderedObjectItems = () => objectItems.toSorted((a, b) => a.order - b.order);
```

Biome (2.5.5)

[error] This variable orderedObjectItems is unused.

(lint/correctness/noUnusedVariables)